scripts/updateVersion.mjs

```diff
@@ -0,0 +1,128 @@
+import semver from "semver";
+import { exec } from "child_process";
+import { readFile, writeFile } from "fs/promises";
+
+const REPO_URL = "https://github.com/nrkno/sofie-core";
```

Comment:
Can we pull this from one of the package.json files?

Reply:
The repository field is only available in to the packages, not in the root or meteor package.json. But I think we should add it.